subsys/bluetooth/audio/unicast_server.c

```diff
@@ -53,6 +53,36 @@ int bt_audio_unicast_server_unregister_cb(const struct bt_audio_unicast_server_c
 	return 0;
 }
 
+int bt_unicast_server_metadata(struct bt_audio_stream *stream,
+			       struct bt_codec_data meta[],
+			       size_t meta_count)
+{
+	struct bt_audio_ep *ep;
+	int err;
+
+
+	if (unicast_server_cb != NULL && unicast_server_cb->metadata != NULL) {
+		err = unicast_server_cb->metadata(stream, meta, meta_count);
+	} else {
+		err = -ENOTSUP;
+	}
+
+	ep = stream->ep;
+	for (size_t i = 0U; i < meta_count; i++) {
+		(void)memcpy(&ep->codec.meta[i], &meta[i],
+			     sizeof(ep->codec.meta[i]));
```

Comment:
Uses the size of the destination.  Is there any chance that the source may be a different size, so that not all (or too much) is copied?

Reply:
`ep->codec.meta[i]` and `&meta[i]` are the same type, so it will be alright :)